### DISCUSSION TITLE
Network Install fails on Ubuntu 22.04 server

### DISCUSSION THREAD

**Discussion by Reaper96D** [Security-Onion-Solutions/securityonion#13925] [2.4] [answered]
### Version

2.4.110

### Installation Method

Network installation on Ubuntu (unsupported)

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

64GB

### Storage for /

328GB

### Storage for /nsm

4TB

### Network Traffic Collection

span port

### Network Traffic Speeds

more than 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I am trying to install the latest SO on a newly built server. It is using a 3Ware 9750-8I card and the SO ISO will not pickup the storage so I am doing the ubuntu 22.04 network install mentioned in the wiki. I got it setup by the tutorial but when I got to run "sudo bash so-setup-network" it allows me to put in the initial information but then once it goes through the scripts, it fails at salt repo. I did some googles and seen this message from salt

"This week, packages from repo.saltproject.io will be migrated to packages.broadcom.com. This means a variety of disruptive changes are being implemented around installing, upgrading, and pinning/locking Salt."

Here is the error during install

"<Start of output>
Ign:1 https://repo.saltproject.io/salt/py3/ubuntu/22.04/amd64/minor/3006.9 jammy InRelease
Hit:2 https://download.docker.com/linux/ubuntu jammy InRelease
Hit:3 http://us.archive.ubuntu.com/ubuntu jammy InRelease
Hit:4 http://us.archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:5 http://us.archive.ubuntu.com/ubuntu jammy-backports InRelease
Hit:6 http://security.ubuntu.com/ubuntu jammy-security InRelease
Ign:1 https://repo.saltproject.io/salt/py3/ubuntu/22.04/amd64/minor/3006.9 jammy InRelease
Ign:1 https://repo.saltproject.io/salt/py3/ubuntu/22.04/amd64/minor/3006.9 jammy InRelease
Err:1 https://repo.saltproject.io/salt/py3/ubuntu/22.04/amd64/minor/3006.9 jammy InRelease
  Could not resolve 'repo.saltproject.io'
Reading package lists...
<End of output>
Forcing exit code to 1
Command failed with exit code 1; will retry in 20 seconds (1 / 150)"

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment, marked as answer]
There is an issue for this https://github.com/Security-Onion-Solutions/securityonion/issues/13926

**Reply by Reaper96D** [reply]
Thank you! I will wait for this change to go into effect.
